src/main/java/net/coderbot/iris/gl/image/ImageLimits.java

```diff
@@ -10,7 +11,7 @@
 
 	private ImageLimits() {
 		if (GL.getCapabilities().OpenGL42) {
```

Comment:
It might make sense to move this code to IrisRenderSystem too since this is a direct GL reference.

Reply:
Done.